src/potential/electrostatics/gammq.cpp

```diff
@@ -0,0 +1,181 @@
+#include <cmath>
```

Comment:
Where did you pull this from? They might require their license at the top of this file.

Reply:
This is code from numerical recipes. It was in our old c++ software, but no reference from where was it taken. i found out it was from NR when I was looking for a vectorizable form of obtaining gamma functions. I will add a comment with probably the citation to the book.

Reply:
Apparently the functions are taken from [here](https://www2.units.it/ipl/students_area/imm2/files/Numerical_Recipes.pdf). They say that it has "limited use". Have you ever used a routine from here? Can we use it? The functions that vlodimir (a former postdoc in our lab) added are slightly modified.

Reply:
If its from the book I would cite the book and add a mention that the post-doc modified them.

Reply:
Will do that.